arrow/VERSION

```diff
@@ -0,0 +1 @@
+13.0.0.1
```

Comment:
I don't think those VERSION files should be part of git tree as they are very dynamic, and keeping them updated will be just polluting git history